app/src/app/components/login/BasicUserAuthForm.tsx

```diff
@@ -40,48 +43,54 @@ export const BasicUserAuthForm = () => {
       navigate("/");
     } catch (error) {
       console.error(error);
-      if (error instanceof ApiError) {
-        if (error.status === 401) {
-          form.setError("email", { message: "Invalid email or password" });
-          form.setError("password", { message: "Invalid email or password" });
+      if (error instanceof ApiError && error.status === HttpStatus.Unauthorized) {
+        if (error.message.includes("must change your password")) {
```

Comment:
Relying on partial message strings feels a bit icky. we could use error codes.

Reply:
yeah felt weird too but had to before when they both were 401.. but have updated so I can throw a 403 as per change suggested